paddle/pten/kernels/impl/matmul_kernel_impl.h

```diff
@@ -176,7 +176,7 @@ void MatMulFunction(const Context& dev_ctx,
                 y_data,
                 x_data,
                 static_cast<T>(flag),
-                Out->mutable_data<T>());
+                Out->data<T>());
```

Comment:
Replace with "mutable_data<T>(dev_ctx.GetPlace())"

Reply:
I found out that the assignment has been performed earlier.